app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RegistrationsController < Devise::RegistrationsController
+  protected
+
+  # After hitting the 'Sign up' page, redirect users here so that they see the
+  # confirmation flash message ("check your email...")
+  def after_inactive_sign_up_path_for(resource)
+    # /auth/users/sign_in
+    new_user_session_path
+  end
+end
```

Comment:
(@dlbucci parody comment :D )

I don't know what these 10 lines do. Our codebase is already bad enough. I'm not sure if adding this meaningless stuff just for a flash message is a good ida.